import-scripts/updated_filename_for_datatype_test.py

```diff
@@ -0,0 +1,534 @@
+meta_drug_treatment_ZSCORE.txt
+meta_drug_treatment_AUC.txt
+meta_RNA_Seq_v2_mRNA_median_all_sample_Zscores.txt
+meta_mRNA_median_all_sample_Zscores.txt
+meta_expression_all_sample_Zscores.txt
+meta_RNA_Seq_mRNA_median_all_sample_Zscores.txt
+meta_RNA_Seq_expression_cpm_all_sample_Zscores.txt
+meta_RNA_Seq_expression_tpm_all_sample_Zscores.txt
+meta_RNA_Seq_expression_capture_all_sample_Zscores.txt
+meta_mRNA_seq_fpkm_capture_all_sample_Zscores.txt
+meta_mRNA_seq_fpkm_polya_all_sample_Zscores.txt
+meta_microbiome.txt
+meta_RNA_Seq_v2_mRNA_median_normals.txt
+meta_RNA_Seq_v2_mRNA_median_Zscores_normals.txt
+meta_RNA_Seq_v2_mRNA_median_all_sample_ref_normal_Zscores.txt
+meta_mutational_signature_contribution_v2.txt
+meta_mutational_signature_pvalue_v2.txt
+meta_mutational_signature_contribution_v3.txt
+meta_mutational_signature_pvalue_v3.txt
+meta_resource_definition.txt
+meta_resource_study.txt
+meta_resource_sample.txt
+meta_phosphoprotein_quantification.txt
+meta_mirna.txt
+meta_mirna_zscores.txt
+meta_mrna_affymetrix_microarray.txt
+meta_mrna_affymetrix_microarray_zscores_ref_diploid_samples.txt
+meta_mrna_affymetrix_microarray_zscores_ref_all_samples.txt
+meta_mrna_agilent_microarray.txt
+meta_mrna_agilent_microarray_zscores_ref_diploid_samples.txt
+meta_mrna_agilent_microarray_zscores_ref_all_samples.txt
+meta_mrna_seq_rpkm.txt
+meta_mrna_seq_rpkm_zscores_ref_diploid_samples.txt
+meta_mrna_seq_rpkm_zscores_ref_all_samples.txt
+meta_mrna_seq_v2_rsem.txt
+meta_mrna_seq_v2_rsem_zscores_ref_diploid_samples.txt
+meta_mrna_seq_v2_rsem_zscores_ref_all_samples.txt
+meta_mrna_seq_v2_rsem_normal_samples.txt
+meta_mrna_seq_v2_rsem_normal_samples_zscores_ref_normal_samples.txt
+meta_mrna_seq_v2_rsem_zscores_ref_normal_samples.txt
+meta_mrna_seq_cpm.txt
+meta_mrna_seq_cpm_zscores_ref_diploid_samples.txt
+meta_mrna_seq_cpm_zscores_ref_all_samples.txt
+meta_mrna_seq_tpm.txt
+meta_mrna_seq_tpm_zscores_ref_diploid_samples.txt
+meta_mrna_seq_tpm_zscores_ref_all_samples.txt
+meta_mrna_seq_fpkm_capture.txt
+meta_mrna_seq_fpkm_capture_zscores_ref_diploid_samples.txt
+meta_mrna_seq_fpkm_capture_zscores_ref_all_samples.txt
+meta_mrna_seq_fpkm_polya.txt
+meta_mrna_seq_fpkm_polya_zscores_ref_diploid_samples.txt
+meta_mrna_seq_fpkm_polya_zscores_ref_all_samples.txt
+meta_mrna_mirna_merged_zscores.txt
+meta_mrna_affymetrix_microarray_merged.txt
+meta_mrna_affymetrix_microarray_merged_zscores_ref_diploid_samples.txt
+meta_mrna_outliers.txt
+meta_rppa_zscores.txt
+meta_protein_quantification_zscores.txt
+meta_cna_foundation.txt
+meta_cna_rae.txt
+meta_cna_consensus.txt
+meta_cna.txt
+meta_linear_cna.txt
+meta_log2_cna.txt
+meta_armlevel_cna.txt
+meta_drug_treatment_auc.txt
+meta_drug_treatment_ic50.txt
+meta_drug_treatment_zscore.txt
+meta_mutations_uniprot_canonical_transcripts.txt
+meta_mutations_uniprot_canonical_transcripts.txt
+meta_mutations_uniprot_canonical_transcripts.txt
+meta_gene_matrix.txt
+meta_gene_panel_matrix.txt
+"""
+
+def filename_maps():
```

Comment:
rename this

Reply:
this has been dropped / transformed into a function filenames_in_use_before_update() which returns a simple string set